src/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,144 @@
+import '../LandingPage/LandingPage.css';
+import headerImg from './LandingPageImages/header-image.svg';
+import brandLogo from './LandingPageImages/Pay IT Forward logo.jpg';
+import featuresImg from './LandingPageImages/features-img.svg';
+import { AiFillCheckCircle } from 'react-icons/ai';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Nav, Navbar, NavDropdown } from "react-bootstrap";
```

Comment:
`NavDropdown` ni kodda hech qayerda ishlatmagansiz. 
```suggestion
import { Nav, Navbar } from "react-bootstrap";
```